modules/lss-missionHelper/missionHelper.user.js

```diff
@@ -726,14 +718,14 @@
         }
       }
       var shortendname = short[0].trim().replace(/\s/g,'').replace(/[(]/g,'').replace(/[)]/g,'').replace(/[-]/g,'').replace("Brandmeldeanlage","");
-      if(shortendname === "Volkslauf"||shortendname === "Dorf/Stadtfest"||shortendname === "BrandsicherheitswachdienstimTheater"||shortendname === "BrandsicherheitswachebeiVolksfest"||shortendname === "AngemeldeteDemonstration"||shortendname === "AbsicherungMusikumzug"||shortendname === "AbsicherungRadrennen"||shortendname === "FeuerprobealarmanSchule"||shortendname === "EntschärfungvonWeltkriegsbombe"||shortendname === "Verkehrsüberwachung"||shortendname === "AbsicherungRockkonzert"||shortendname === "Schwertransport" || shortendname ==="HasentreffeninOstereistedt" || shortendname ==="FußballBundesligaRisikospiel" || shortendname === "Concert"||shortendname === "SoccerGame"||shortendname === "BaseballGame" || shortendname === "Voetbalwedstrijd"||shortendname === "Voetbalwedstrijd,risicowedstrijd"||shortendname === "Hardloopevenement"||shortendname === "Begeleidingorkest"||shortendname === "Begeleidingwielrenners"||shortendname === "Testbrandalarmschool"||shortendname === "Open dag,klein"||shortendname === "Open dag,groot"){
+      if(shortendname === "Volkslauf"||shortendname === "Dorf/Stadtfest"||shortendname === "BrandsicherheitswachdienstimTheater"||shortendname === "BrandsicherheitswachebeiVolksfest"||shortendname === "AngemeldeteDemonstration"||shortendname === "AbsicherungMusikumzug"||shortendname === "AbsicherungRadrennen"||shortendname === "FeuerprobealarmanSchule"||shortendname === "EntschärfungvonWeltkriegsbombe"||shortendname === "Verkehrsüberwachung"||shortendname === "AbsicherungRockkonzert"||shortendname === "Schwertransport" || shortendname ==="HasentreffeninOstereistedt" || shortendname ==="FußballBundesligaRisikospiel" || shortendname === "Concert"||shortendname === "SoccerGame"||shortendname === "BaseballGame" || shortendname === "Voetbalwedstrijd"||shortendname === "Voetbalwedstrijd,risicowedstrijd"||shortendname === "Hardloopevenement"||shortendname === "Begeleidingorkest"||shortendname === "Begeleidingwielrenners"||shortendname === "Testbrandalarmschool"||shortendname === "Opendag,klein"||shortendname === "Opendag,groot"){
         return "nope";
       }
       return shortendname;
     }
 
-    var missionname = Normalize(document.getElementById("missionH1"));
-    if(missionname != "nope"){
+    var missionname = normalize(document.getElementById("missionH1"));
+    if(missionname !== "nope"){
```

Comment:
ja ich bin grad dabei deinen anderen vorschlag umzusetzen und schau mal was sich machen lässt. Aber bis dato hat es ohne probs gefunzt, wollte es jetzt ebend nicht einfach pullen in den dev ^^. einfach aufgrund das ich da ein paar sachen verändert habe die zwar bei mir ohne probs laufen aber vlt bei euch anderen net